utils/constants.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+export const BASE_API_PATH = "/api/ism";
+export const NODE_API = Object.freeze({
+  SEARCH: `${BASE_API_PATH}/_search`,
+  GET_INDICES: `${BASE_API_PATH}/_indices`,
+  ADD_POLICY: `${BASE_API_PATH}/addPolicy`,
+  GET_POLICIES: `${BASE_API_PATH}/policies`,
+  PUT_POLICY: `${BASE_API_PATH}/policies`,
+  GET_POLICY: `${BASE_API_PATH}/policies`,
+  DELETE_POLICY: `${BASE_API_PATH}/policies`,
+  GET_MANAGED_INDICES: `${BASE_API_PATH}/managedIndices`,
+  GET_MANAGED_INDEX: `${BASE_API_PATH}/managedIndices`,
```

Comment:
why is there two of same path but named differently? `GET_MANAGED_INDICES` vs `GET_MANAGED_INDEX`?

Reply:
Just did a quick conversion, updated to reflect the actual path.